src/euphorie/client/docx/compiler.py

```diff
@@ -621,27 +647,43 @@ def set_cell_actions(self, cell, risk):
             if action.get('prevention_plan', None):
                 paragraph = cell.add_paragraph(style="Measure Indent")
                 run = paragraph.add_run()
-                run.text = u"Actions : "
+                run.text = api.portal.translate(
+                    _(u"report_actions", default=u"Actions:")
+                )
                 run.underline = True
                 run = paragraph.add_run()
+                run.text = u" "
+                run = paragraph.add_run()
                 run.text = _simple_breaks(action['prevention_plan'])
             if action.get('requirements', None):
                 paragraph = cell.add_paragraph(style="Measure Indent")
                 run = paragraph.add_run()
-                run.text = u"Compétences requises : "
+                run.text = api.portal.translate(
+                    _(u"report_competences", default=u"Required expertise:")
+                )
                 run.underline = True
                 run = paragraph.add_run()
+                run.text = u" "
+                run = paragraph.add_run()
                 run.text = _simple_breaks(action['requirements']),
 
             if action.get('responsible', None):
                 paragraph = cell.add_paragraph(
-                    u"Responsable: {}".format(action['responsible']),
+                    u"{0} {1}".format(
+                        api.portal.translate(
+                            _(u"report_responsible", default=u"Responsible:")
+                        ),
+                        action['responsible']),
                     style="Measure Indent"
                 )
                 paragraph.runs[0].italic = True
             if action.get('planning_start', None):
                 paragraph = cell.add_paragraph(
-                    u"Date de fin: {}".format(action['planning_start']),
+                    u"{0} {1}".format(
+                        api.portal.translate(
+                            _(u"report_end_date", default=u"To be done by:")
+                        ),
+                        action['planning_start']),
```

Comment:
It would be better to add an i18n name in the translated string:
```
api.portal.translate(
    _(u"report_end_date", default=u"To be done by: ${date}", mapping={"date": action['planning_start'])})
)
```
Untested ^

Reply:
ok, done